### PR TITLE
Fix lazy initialized actors exception

### PR DESCRIPTION
**Pull request by sarakhmen**
Applying the next [improvement](https://prnt.sc/5TRAgTZ2JJOy)

### CODE REVIEW

File: src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java [hunk 1]
```diff
@@ -107,4 +111,13 @@ private void verifyCreateMovieWorks(MovieDao movieDao, Movie movie, Long expecte
         Assert.assertEquals(expectedId, actual.getId());
         Assert.assertEquals(movie.getTitle(), actual.getTitle());
     }
+
+    private void initializeActors(Movie movie){
+        try(Session session = getSessionFactory().openSession()){
+            session.refresh(movie);
+            Hibernate.initialize(movie.getActors());
+        }catch (Exception e){
```
Comment (boroda4436) [inline src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java:119]:
```suggestion
        } catch (Exception e) {
```

File: src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java [hunk 2]
```diff
@@ -107,4 +111,13 @@ private void verifyCreateMovieWorks(MovieDao movieDao, Movie movie, Long expecte
         Assert.assertEquals(expectedId, actual.getId());
         Assert.assertEquals(movie.getTitle(), actual.getTitle());
     }
+
+    private void initializeActors(Movie movie){
+        try(Session session = getSessionFactory().openSession()){
+            session.refresh(movie);
+            Hibernate.initialize(movie.getActors());
```
Comment (boroda4436) [inline src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java:118]: `Hibernate.initialize` is not the best approach to fix LazyInitializationException. Can we use join fetch here?

File: src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java [hunk 3]
```diff
@@ -107,4 +111,13 @@ private void verifyCreateMovieWorks(MovieDao movieDao, Movie movie, Long expecte
...
+        }catch (Exception e){
+            throw new RuntimeException(e);
```
Comment (boroda4436) [inline src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java:120]: add some informative message as a first param to the constructor

**Review by boroda4436 — CHANGES_REQUESTED** [review]

**Review by boroda4436 — APPROVED** [review]

File: src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java [hunk 4]
```diff
@@ -107,4 +108,16 @@ private void verifyCreateMovieWorks(MovieDao movieDao, Movie movie, Long expecte
         Assert.assertEquals(expectedId, actual.getId());
         Assert.assertEquals(movie.getTitle(), actual.getTitle());
     }
+
+    private Optional<Movie> getMovie(Long id) {
```
Comment (Nick97-git) [inline src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java:112]:
```suggestion
    private Optional<Movie> getMovieById(Long id) {
```

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**